docs/TESTS.md

```diff
@@ -41,6 +41,22 @@ export default class Bob {
 }
 ```
 
+## Understanding Skip Tests
+
+The skip method instructs the test suite to not run a test, this function could be used also under the aliases: `it.skip(name, fn) or xit(name, fn) or xtest(name, fn)`
+
+- Why they are skipped ? 
+So as to enable users to concentrate on one test at a time and enable one by one as they evolve the solution.
+
+- How to enable them ? 
+Change xtest to test.
+
+```javascript
+test('title cased phrases', () => {
+  expect(Acronyms.parse('Portable Network Graphics')).toEqual('PNG');
+});
+```
+
```

Comment:
Can you move this section above modules, I think it would be better context there after `npm run watch`.

Also the line below this (babel documentation) is a part of modules section, not skipping tests section.